### PR TITLE
tests: add test cases for int64 field, capital letter field, map pagination

### PR DESCRIPTION
**Pull request by georgiyekkert**
Add tests for:
1) Pagination with map response
2) A field with format: int64
3) A field that starts with a capital letter, like IPAddress

### CODE REVIEW

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java [hunk 1]
```diff
@@ -116,4 +121,28 @@ public void testPaginationIterating() {
...
+            .setMaxResults(2)
+            .build();
+    AcceleratorTypesClient.AggregatedListPagedResponse response =
+        acceleratorTypesClient.aggregatedList(aggregatedListAcceleratorTypesRequest);
+    boolean presented = false;
+    for (Map.Entry<String, AcceleratorTypesScopedList> entry : response.iterateAll()) {
+      if (entry.getKey().equals("zones/" + DEFAULT_ZONE)) {
+        for (AcceleratorType type : entry.getValue().getAcceleratorTypesList()) {
+          if (type.getName().equals("nvidia-tesla-t4")) {
+            presented = true;
```
Comment (vam-google) [inline google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java:140]: did you mean "present"?

**Review by vam-google — COMMENTED** [review]
LGTM but please add in the description which specific new tests you are adding here

File: google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java (hunk 1 above)
Comment (georgiyekkert) [reply google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITPaginationTest.java:140]: I used "presented" as a short form of "is presented in the list"

**Review by vam-google — APPROVED** [review]
ДПЕЬ

**Pull request merged**